imputation/src/mean/mean_test.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+    "testing"
+)
+
+func TestMean(t *testing.T) {
+    // test 1 empty series
```

Comment:
Unlike all the programming languages I know, Go prefers table-driven tests.  Q.v., https://dave.cheney.net/2019/05/07/prefer-table-driven-tests.  Let's add a todo to update this code to be more table driven.

```
TODO(eftekhari-mhs): Refactor to table-driven tests.
```